boards/arm/nucleo_f429zi/nucleo_f429zi.dts

```diff
@@ -61,6 +61,7 @@ arduino_serial: &usart6 {};
 
 &spi1 {
 	status = "ok";
+	cs-gpios = <&gpioa 4 0>;
```

Comment:
cs-gpio depends on the way you add hardware to the board and is specific to your need, we cannot take this modification.
Please remove